acoustid.py

```diff
@@ -179,10 +180,14 @@ def _api_request(url, params, timeout=None):
         "Content-Type": "application/x-www-form-urlencoded"
     }
 
+    params['meta'] = '+'.join(params['meta'])
```

Comment:
I'd say this should really joined by `' '`. It doesn't matter if it gets encoded as `+` or `%20`.

Reply:
Thank you for your input! That's right, it doesn't matter. But using `+` fits better to the AcoustID Web Service documentation. IMO would this improve the maintainability, 'cause the concated string fits to the documentation. This leads to a better detection of at least an error in the formatting of the `meta` parameter.

Reply:
@lalinsky does make a good point here; in HTTP, `+` and `%20` are just different ways to encode a space (so servers should be bound to treat them the same).

Reply:
I replaced `+` with whitespace and moved this into the try-catch below. Also implemented check with `isinstance()`.